includes/Renderer/Story/HTML.php

```diff
@@ -54,15 +56,24 @@ class HTML {
 	 */
 	protected $document;
 
+	/**
+	 * Experiments instance.
+	 *
+	 * @var Experiments Experiments instance.
+	 */
+	private $experiments;
+
 	/**
 	 * HTML constructor.
 	 *
 	 * @since 1.0.0
 	 *
-	 * @param Story $story Story object.
+	 * @param Story       $story Story object.
+	 * @param Experiments $experiments Experiments instance.
 	 */
-	public function __construct( Story $story ) {
-		$this->story = $story;
+	public function __construct( Story $story, Experiments $experiments ) {
+		$this->story       = $story;
+		$this->experiments = $experiments;
```

Comment:
Instead of injecting like this. Why not trying this 

https://github.com/google/web-stories-wp/blob/349bfc194416097720dae8bf2f408dd9a819f5cc/includes/Renderer/Stories/Renderer.php#L165-L170

Reply:
`Injector::make` creates a new instance, no? `Services::get( 'experiments' )` would get the current instance. I might have tried that, but it might not have worked, don't remember.

The current approach works fine, don't see big issues with it. It's temporary anyway until we remove the feature flag.